manual/development-process/access-integration-db.md

```diff
@@ -0,0 +1,78 @@
+#Accessing the Integration database
+
+Connecting to the Integration database must be done through an AWS Bastion host, below are the steps detailing how to do this
+
+##1. Downloading and installing the Session Manager plugin
+First you need to download and install the AWS Session Manager plugin
+
+   * [Instructions for Mac (Step 1 to 3)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-macos)
+   * [Instructions for Linux (Step 1 to 2)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-linux)
+   * [Instructions for Ubuntu Server (Step 1 to 2)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-debian)
+
+You can check if it worked by running  `session-manager-plugin`, which should show you the message
+
+`The Session Manager plugin is installed successfully. Use the AWS CLI to start a session.`
+
+##2. Obtaining the Instance ID for the Bastion instance
```

Comment:
Could you or Sam add some instructions for starting and stopping the bastion host? It's not a permanent instance.

Reply:
You need to run this job https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Bastion%20Deploy/build
Choose which stage you want and either apply or destroy to bring it up or down. You can add an ssh key if you're creating a bastion if you want to create an ssh tunnel but it's optional.
Let me know if you want me to write this up properly but that's the general gist.